docs/01-Introduction.md

```diff
@@ -1,6 +1,89 @@
 # 1.0 - Introduction
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI? 
+LORIS-MRI is the backbone of the imaging component that makes up LORIS.
+ It is maintained in a separate repository so that it can be installed
+ on the file server and separated from the web server.
+These documents assume you have some knowledge regarding LORIS and
+a functioning installation. For information regarding LORIS itself,
+please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is a set of libraries responsible for the insertion,
+organization, and archival of uploaded imaging datasets.
+ It expects an uploaded, compressed file containing
+a [DICOM][2] study composed of several DICOM files. These DICOM files
+ will be archived on the server and converted to [MINC][3] and [NIfTI][4]
```

Comment:
@davidblader 
please add the word optionally to NIfTI (it is optional depending on project config settings)